doc/api/fs.md

```diff
@@ -368,6 +368,24 @@ changes:
     {fs.Stats} object should be `bigint`. **Default:** `false`.
 * Returns: {Promise} Fulfills with an {fs.Stats} for the file.
 
+#### `filehandle.stream(dst, [options])`
```

Comment:
To be honest, I'd be happier with an API that would work with the existing streams API and pipeline utility. Specifically, something like:

```js
pipeline(filehandle.readable(), dst, () => {});

pipeline(src, filehandle.writable(), () => {});
```

Where the return value of `filehandle.readable()` is essentially an `fs.ReadStream` and `filehandle.writable()` is an `fs.WriteStream`.

If we did want an API like this, the I would more expect `filehandle.pipe(dst, [options])` or `filehandle.pipeTo(dst, [options])`.

Reply:
> Specifically, something like:

That kind of removes the point of this though. Will have the same performance as just doing `fs.createReadStream`.

Reply:
Performance is not the only concern, however... a consistent and meaningful API that is not confusing is as equally important. What I'd rather see is effort into making `fs.ReadStream` and `fs.WriteStream` faster and for there to be a shared code path here so that overall maintainability, consistency, *and* performance are all improved.